include/exadg/solvers_and_preconditioners/solvers/iterative_solvers_dealii_wrapper.h

```diff
@@ -282,7 +282,9 @@ class SolverGMRES : public SolverBase<VectorType>
     }
 
     AssertThrow(std::isfinite(solver_control.last_value()),
-                dealii::ExcMessage("Solver contained NaN of Inf values"));
+                dealii::ExcMessage("Convergence value of last iteration step "
+                                   "for which check was called by the solver "
```

Comment:
I do not understand what you mean by "for which check was called" and "convergence value".

I think the present PR should just fix the setup-problem (potentially with a single new test case).

I found three instances of this exc message in that file. So why did you change only some of them? Let's keep these changes out of the present PR.

Reply:
This is from the deal.II docu:
"Return the convergence value of last iteration step for which check was called by the solver."
(https://www.dealii.org/current/doxygen/deal.II/classSolverControl.html#a4a3b4da9834a3a5f1da12fa8fcc074f0)
Acc. to the docu, it is not always the residual that is checked
"To this end, the virtual function [check()](https://www.dealii.org/current/doxygen/deal.II/classSolverControl.html#a56fdf4c8d099edaca6fad443d7c76649) is called in each iteration with the current iteration step and the value indicating convergence (usually the residual)."
Within ExaDG, I think residual would actually be correct.

I only changed these because I did not see the other one.
Initially, I only saw the typo.